Tasks/PublishSymbols/PublishSymbolsTask.ps1

```diff
@@ -0,0 +1,44 @@
+[CmdletBinding()]
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+try {
+    $symbolServerType = Get-VstsInput -Name 'SymbolServerType' -Require
+
+    if ($SymbolServerType -eq "FileShare")
+    {
+        & "$PSScriptRoot\PublishSymbols.ps1"
+    }
+    elseif ($symbolServerType -eq "TeamServices")
```

Comment:
Is indexing not an option when publishing to TeamServices? I don't see a visibility rule on the indexing inputs in the task.json. I don't see a code path where indexing could get called when publishing to TeamServices.